cranelift/assembler-x64/meta/src/generate/format.rs

```diff
@@ -15,15 +15,12 @@ impl dsl::Format {
     /// (TODO).
     #[must_use]
     pub fn generate_att_style_operands(&self) -> String {
-        let mut ordered_ops: Vec<_> = self
+        let ordered_ops: Vec<_> = self
             .operands
             .iter()
+            .rev()
```

Comment:
Huh... let's try this but I'm not sure.

Reply:
Heh agreed! I figure we'll need to add more heuristics here eventually to figure out how to print and still match att/capstone ordering.